step-templates/sitecore-ship-deploy.json

```diff
@@ -0,0 +1,82 @@
+    "Octopus.Action.CustomScripts.Deploy.ps1": "",
+    "Octopus.Action.CustomScripts.PostDeploy.ps1": ""
+  },
+  "Parameters": [
+    {
+      "Id": "8be53044-a688-48db-bc0c-d7c29c9d5993",
+      "Name": "SiteUrl",
+      "Label": "Site Url",
+      "HelpText": "Url for the site you are installing the .update package on (ex: _http://mysite.dev_)",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "d4bd2493-7325-43b5-9e41-6a5c2fd03434",
+      "Name": "ConnectionTimeOutInSeconds",
+      "Label": "Connection Timeout",
+      "HelpText": "Connection Timeout in seconds (ex: _300_)",
+      "DefaultValue": "300",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "a3365b81-a8c2-4554-ba29-0fca2d52a2bd",
+      "Name": "Username",
+      "Label": "Username",
+      "HelpText": "If your Ship endpoint requires authentication, insert Username here",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "0151a6e4-f79b-4847-811c-eb6c43a40838",
+      "Name": "Password",
+      "Label": "Password",
+      "HelpText": "If your Ship endpoint requires authentication, insert Password here",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "b03b95c5-aabc-4c5f-9585-cbaa0c93f621",
+      "Name": "PackageID",
+      "Label": "Package ID",
+      "HelpText": "The ID of the package (from the builtin feed) you want to deploy",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    }
+  ],
+  "Category": "sitecore",
+  "LastModifiedBy" : "svandenbush@geekhive.com",
```

Comment:
If you update this to your :octocat: username, it will be linked correctly on the Library site